Sources/Misc/Concurrency/OperationDispatcher.swift

```diff
@@ -108,6 +109,8 @@ private extension Delay {
         case .none: return 0
         case .`default`: return 0
         case .long: return Self.maxJitter
+        case .timeInterval(let timeInterval):
+            return max(timeInterval, 0)
```

Comment:
I can't think of any instances where the timeInterval would be <0 in the scope of this PR, but this is just defensive programming in case someone ever tries to pass in a negative `TimeInterval` as a delay in the future